### PR TITLE
ServiceMax RTL changes, proper version bump

### PR DESCRIPTION
**Pull request by joshsylvester**
- Fixed Dropdown icon spacing issue when the rendering direction is "rtl" Right-To-Left.

### CODE REVIEW

**Comment by joshsylvester** [comment]
@evanjd @arongraham 2 issues:
1. I'm submitting this from the ServiceMax fork, not my own. it's saying that i don't have a CLA signed, so i signed another one as the org. still says that it's not signed. any ideas? do i need to recreate from my personal account?
2. Locally - these changes are *not* visible. I can make changes to **scss**, run `gulp serve` / `gulp sass`, but the css file that is generated `css/px-dropdown-styles.html` is never included or downloaded, so the changes cannot be tested. I've manually made the same changes to the DOM via the devtools, so i know that it works, but i can't confirm. any ideas on how to test the component's **scss** changes?

**Comment by evanjd** [comment]
> 1. I'm submitting this from the ServiceMax fork, not my own. it's saying that i don't have a CLA signed, so i signed another one as the org. still says that it's not signed. any ideas? do i need to recreate from my personal account?

Commits from @shao are in this PR, and they have not signed the CLA. You can get around this by squashing their commit into another with rebase.

> 2\. Locally - these changes are _not_ visible. I can make changes to **scss**, run `gulp serve` / `gulp sass`, but the css file that is generated `css/px-dropdown-styles.html` is never included or downloaded, so the changes cannot be tested. I've manually made the same changes to the DOM via the devtools, so i know that it works, but i can't confirm. any ideas on how to test the component's **scss** changes?

How annoying is this! Your changes won't be visible on the demo page because it imports px-dropdown from bower_components (via px-demo, which depends on px-dropdown). You _should_ be able to test your changes from [simple-demo ](https://github.com/predixdesignsystem/px-dropdown/blob/master/demo/simple_demo.html) though as that imports the repo's copy of px-dropdown.

**Comment by joshsylvester** [comment]
@evanjd Re: 1. I've tried a few dozen times to rebase, but unfortunately, i'm not very good at it, and i can't get it to work. I've reached out to @shao to ask him to sign the CLA instead.

Re: 2. Thank you for the suggestion of the simple demo - I've been able to confirm our changes on that demo and prove that they work as expected.

File: sass/px-dropdown.scss [hunk 1]
```diff
@@ -58,11 +58,17 @@ $inuit-enable-btn--primary: true;
...
   --iron-icon-width: 1.06667rem;
   --iron-icon-height: 1.06667rem;
   margin-left: $inuit-base-spacing-unit--tiny;
+  /*
+    These margins are for pages that are rendered from "Right-To-Left" or "Left-To-Right"
+    If the writing direction is left-to-right, this value overrides margin-left.
+    If the writing direction is right-to-left, this value overrides margin-right.
+  */
+  -webkit-margin-start: $inuit-base-spacing-unit--tiny;
+  -moz-margin-start: $inuit-base-spacing-unit--tiny;
```
Comment (evanjd) [inline sass/px-dropdown.scss]: What about IE and Edge?

Comment (joshsylvester) [reply sass/px-dropdown.scss]: no support for these properties in IE/Edge - the content doesn't look bad, the only issue is that the carat for the dropdown is not spaced away from the text; it's still usable.

`normal dropdown in IE/Edge`
dropdown-ie-edge

`RTL dropdown in ie/edge`
rtl-dropdown-ie-edge

**Comment by joshsylvester** [comment]
I'm going to close this PR and recreate by manually making the changes needed to get around the CLA issue.

**Pull request closed without merging**